data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2652,7 +2652,7 @@
 
         [message]
             speaker=Malifor
-            message= _ "You invaded my kingdom, drove me from my mines, raided my dungeon, and plundered my treasury. Your audacity ends here!"
+            message= _ "You invaded my kingdom, drove me from my mines, raided my dungeon, and plundered my treasury. Your impertinence ends here!"
```

Comment:
Both seem similar to me. Maybe a `# po:` comment would make it clearer - say something about why it should be one rather than the other, about which specific detail of Malifor's opinion of the protagonists is driving the choice.

Reply:
Real quick, what exactly is a # po: comment?

I did it because while dictionarily they sort of mean the same thing. Audacity has a positive sort of meaning, while impertinence seems a more negative word.

Reply:
While I can imagine audacity being used in a positive sense, I consider it a neutral term - I wouldn't think of it in the positive sense here. Impertinence certainly has a stronger negative connotation to my reading, and it made sense in the context of Malifor speaking, so I let this one pass.

Reply:
> Real quick, what exactly is a # po: comment?

WML comments beginning with `po:` that are on the line immediately before a translatable string are copied as notes for the translators. [Details on the Wiki.](https://wiki.wesnoth.org/GettextForWesnothDevelopers#Notes_to_the_translators)

Currently there's a discussion in Discord about whether `#po:` or `# po:` is preferred, but currently the space is optional.

Reply:
Audacity isn't only used in a positive way so I think this is unnecessary.

Reply:
I didn't see this one. For some reason. I'll change it, but I don't agree.